api/src/main/java/io/opentelemetry/trace/Span.java

```diff
@@ -304,6 +306,43 @@
    * @since 0.1.0
    */
   interface Builder {
+    /**
+     * Sets the parent {@code Span} to use. If not set, {@code Tracer.getCurrentSpan()} will be used
+     * as parent.
```

Comment:
Does it worth commenting that the Tracer.getCurrentSpan() will be called during the build() time if no parent was specified? Otherwise some implementations may read it at the Builder creation time and reusing a SpanBuilder may generate undefined behaviors.

Reply:
Oh, yeah, good clarification.

Reply:
Not sure I see any update about this.